js/lightbeam.js

```diff
@@ -1,10 +1,14 @@
 const lightbeam = {
   websites: {},
+  dataGatheredSince: '',
```

Comment:
I would make this null and at presentation time turn this into a blank string to be clear the difference between the data itself and the presentation.